napari_clusters_plotter/_plotter.py

```diff
@@ -362,36 +362,59 @@ def run(self, properties, plot_x_axis_name, plot_y_axis_name, plot_cluster_name=
         if plot_cluster_name is not None and plot_cluster_name != "label" and plot_cluster_name in list(properties.keys()):
             self.cluster_ids = properties[plot_cluster_name]
 
-            color = ['#ff7f0e', '#1f77b4', '#2ca02c', '#d62728', '#9467bd', '#8c564b', '#e377c2', '#7f7f7f', '#bcbd22',
-                     '#17becf']
+            # get long colormap from function
+            colors = get_nice_colormap()
+            
             self.graphics_widget.pts = self.graphics_widget.axes.scatter(
                 self.data_x,
                 self.data_y,
-                c=[color[int(x)] for x in self.cluster_ids],
+                c=[colors[int(x)] for x in self.cluster_ids],
```

Comment:
You have a long list of colors but maybe there should be a cycle operation 
```suggestion
                c=[colors[int(x) % len(colors)] for x in self.cluster_ids],
```